README.md

```diff
@@ -21,276 +21,14 @@ GP2GP 2.2b producer, or those wishing to decommission their existing producer, m
-            "fileName": "54E94A29-B0CC-4CD6-86B0-B21C9C0CA894.doc",
-            "originalDescription": "Referral for further care (22-Dec-2020)"
-        },
-        {
-            "identifier": [
-                {
-                    "system": "https://EMISWeb/A82038",
-                    "value": "D7AF52BA-79BA-4AF8-9010-F0C2DF916CEC"
-                }
-            ],
-            "fileStatus": "ORIGINAL_FILE",
-            "fileName": "629BF3F7-C71F-49D7-8FCD-AEE16C11AFBD.doc",
-            "originalDescription": "Referral for further care (22-Dec-2020)"
-        },
-        {
-            "identifier": [
-                {
-                    "system": "https://EMISWeb/A82038",
-                    "value": "D7AF52BA-79BA-4AF8-9010-F0C2DF916CEC"
-                }
-            ],
-            "fileStatus": "ORIGINAL_FILE",
-            "fileName": "2F73A243-0F94-4683-B3F9-727CE7780815.doc",
-            "originalDescription": "Referral for further care (22-Dec-2020)"
-        }
-    ],
-    "migrationLog": [
-        {
-            "received": "2023-07-24T10:42:18.591Z",
-            "conversationClosed": null,
-            "errors": null,
-            "messageRef": "ED2FB0DD-52AE-4EBC-B596-3BCA9CF5B272"
-        },
-                {
-            "received": "2023-07-24T10:42:19.176Z",
-            "conversationClosed": null,
-            "errors": null,
-            "messageRef": "5068BAE1-7228-4E59-B3AD-92FA5CBDE4AC"
-        },
-                {
-            "received": "2023-07-24T10:42:19.467Z",
-            "conversationClosed": null,
-            "errors": null,
-            "messageRef": "A9E37B68-07F4-43A5-889E-645D6681CE68"
-        },
-        {
-            "received": "2023-07-24T10:42:22.611Z",
-            "conversationClosed": "2023-07-24T10:42:22.611Z",
-            "errors": [
-                {
-                    "code": "99",
-                    "display": "Unexpected condition."
-                }
-            ],
-            "messageRef": "E600D1D4-D5BE-4D0D-8080-F7F6188B0548"
-        }
-    ],
-    "migrationStatus": "FAILED_INCUMBENT",
-    "originalRequestDate": "2023-07-24T10:42:05.757Z",
-    "fromAsid": "918999198738",
-    "toAsid": "200000000359"
-}
-```
-#### Failed by adaptor, due to a PATIENT_NOT_FOUND error from the GP Connect provider's migrate structured endpoint
-```json
-
-{
-    "attachmentStatus": [],
-    "migrationLog": [],
-    "migrationStatus": "FAILED_NME",
-    "originalRequestDate": "2023-07-26T15:36:22.284Z",
-    "fromAsid": "918999198738",
-    "toAsid": "200000000359"
-}
-```
-</details>
-
-## How to query the Requests endpoint
-
-An API is provided to query the status of all transfers to an incumbent.
 Any "in-progress" transfers are excluded from this list, but become available once they either succeed or fail.
 
-Requests can be made to the following endpoint, where each attribute within the JSON POST body is an optional filter criteria.
```

Comment:
Related to my comments below about `required`, this comment reaffirms my understanding which is that the filtering is optional.

Reply:
ok, I see, make sense, I will look into it and adjust it

Reply:
I think as object it is required but EhrStatusRequestQuery object has fields which are not mandatory and can be empty.